spring-cloud-sleuth-api/src/main/java/org/springframework/cloud/sleuth/Span.java

```diff
@@ -99,6 +99,17 @@ default Span remoteServiceName(String remoteServiceName) {
 		return this;
 	}
 
+	/**
+	 * Sets the remote url on the span.
+	 * @param ip remote ip
+	 * @param port remote port
+	 * @return this span
+	 * @since 3.1.0
+	 */
+	default Span remoteUrl(String ip, int port) {
```

Comment:
Should we call it `remoteUrl`?
Brave calls it `remoteIpAndPort`.

Reply:
I know. I thought that `remoteUrl` sounds better. I don't have a problem to name it in the same way as Brave does it